js/src/components/google-combo-account-card/claim-ads-account/claim-ads-account.scss

```diff
@@ -0,0 +1,29 @@
+.gla-claim-ads-account-section {
+	padding-top: 0;
+}
+
+.gla-claim-ads-account-box {
+	border: 1px solid $gray-300;
+	font-size: $gla-font-small;
```

Comment:
Setting font size is unnecessary as all children have been set their font size.

Reply:
👍